client/src/components/Signup.js

```diff
@@ -887,6 +904,30 @@ export default class Signup extends React.Component {
                             this.getAlumniFields() :
                             this.getStudentFields()
                         }
+                        <Divider/>
+                        <Form.Field
+                            required={true}
+                            style = {fieldStyle}
+                        >
+                            <label> Terms of Agreements</label>
+                            <Button
+                                primary color="blue"
+                                type="button"
+                                onClick={() => {this.setState({termsModalOpen: true})}}
```

Comment:
This is fine, but the callback may be simplified by having the onClick just take this.handleTermsModal